### DISCUSSION TITLE
Makita Simulations Missing Title

### DISCUSSION THREAD

**Discussion by kanisanne** [asreview/asreview#1930] [Q&A]
While running simulations with Makita, I ran into issues when I tried putting in content only for the abstract portion and leaving the title blank. (Why) is the title section mandatory and is there a difference to the simulation process between putting content only in the title vs abstract section?

**Comment by jteijema** [comment]
Hi @kanisanne, could you share the exact error message you're getting? I'd like to take a look!

**Reply by kanisanne** [reply]
I took down the following two error messages when tinkering with the format of in/exclusion criteria as prior knowledge (text in the abstract column) instead of (ir)relevant records: 
- `ERROR: Dataset data\prior_Appenzeller-Herzog_2019_clean.csv contains unlabeled record at row 1.
Title: ''`
- `ERROR: File supplied without 'abstract' or 'title' fields.`
This prompted me to think that the issue was the empty title, and trying placeholder titles removed these initial errors. 
However, when moving further I realised that it is possible to produce simulations without a title, there was seemingly just a different (Windows) bug I was experiencing.
Moving the entire file structure out of OneDrive made it possible to finish running simulations with empty title columns.

**Comment by Rensvandeschoot** [comment]
As noted in the [ASReview documentation](https://asreview.readthedocs.io/en/latest/data_format.html):

> "Title, Abstract: Each record (i.e., entry in the dataset) should hold metadata on a paper. Mandatory metadata are only title or abstract. If both title and abstract are available, the text is combined and used for training the model. If the column title is empty, the software will search for the next column primary_title and the same holds for abstract and abstract_note."

This means the software requires at least one field—either the title or the abstract—to contain text. Both fields are combined into a single text input for the model. If both fields are empty, the record is skipped as there is no text for the model to process. 

Regarding your question about whether the title is specifically mandatory, it likely isn’t required in isolation, but rather some text input is necessary. It’s possible that implementing the requirement to allow "at least one field to be filled" (title or abstract) was simpler with this structure. However, I can’t recall any specific reason why the title itself must always contain text. (ping: @J535D165 )

To confirm this, could you try creating a small example dataset where the title column is entirely empty but the abstract column contains text? This would help demonstrate whether the system can proceed without issues as long as the abstract is filled, and to improve the error message.
